build.gradle

```diff
@@ -434,6 +436,25 @@ subprojects {
     }
     logger.lifecycle "Tests for subproject ${project.name} will be skipped."
   }
+
+  dependencies.constraints {
+    add("compile", "com.google.guava:guava") {
+      attributes {
+        attribute(Attribute.of("org.gradle.jvm.environment", String),"standard-jvm")
+//        attribute(TargetJvmEnvironment.TARGET_JVM_ENVIRONMENT_ATTRIBUTE,
+//                objects.named(TargetJvmEnvironment.STANDARD_JVM))
+      }
+    }
+  }
+
+  configurations.configureEach {
```

Comment:
If we need to keep this in the build.gradle we might want to add comments here and above about what this does and/or why it's needed